common/install_cuda.sh

```diff
@@ -361,6 +381,38 @@ function prune_112 {
     rm -rf $CUDA_BASE/libnvvp $CUDA_BASE/nsightee_plugins $CUDA_BASE/nsight-compute-2020.3.0 $CUDA_BASE/nsight-systems-2020.4.3
 }
 
+function prune_113 {
+    echo "Pruning CUDA 11.3 and CuDNN"
+    #####################################################################################
+    # CUDA 11.3 prune static libs
+    #####################################################################################
+    export NVPRUNE="/usr/local/cuda-11.3/bin/nvprune"
+    export CUDA_LIB_DIR="/usr/local/cuda-11.3/lib64"
+
+    export GENCODE="-gencode arch=compute_35,code=sm_35 -gencode arch=compute_50,code=sm_50 -gencode arch=compute_60,code=sm_60 -gencode arch=compute_70,code=sm_70 -gencode arch=compute_75,code=sm_75 -gencode arch=compute_80,code=sm_80 -gencode arch=compute_86,code=sm_86"
+    export GENCODE_CUDNN="-gencode arch=compute_35,code=sm_35 -gencode arch=compute_37,code=sm_37 -gencode arch=compute_50,code=sm_50 -gencode arch=compute_60,code=sm_60 -gencode arch=compute_61,code=sm_61 -gencode arch=compute_70,code=sm_70 -gencode arch=compute_75,code=sm_75 -gencode arch=compute_80,code=sm_80 -gencode arch=compute_86,code=sm_86"
```

Comment:
Is there a reason we keep sm_61 here?

Reply:
iirc there was an error when we actually went through and removed it, cc @ngimel @ptrblck